database/index.js

```diff
@@ -1,6 +1,7 @@
 const mongoose = require('mongoose');
+
 const mongoUri = 'mongodb://localhost/infoModule';
 
 const db = mongoose.connect(mongoUri);
```

Comment:
@wwong007 @iamjaytong Probably a good idea to handle errors that might come from connecting to the database. Might also want some kind of console log indicating a successful connection has been made (if indeed it has)